blocks/card-social-post/src/CardSocialPost.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {
+  Avatar,
+  AvatarGroup,
+  Card,
+  CardContent,
+  Stack,
+  Typography,
+} from "@mui/joy";
+
+export default function CardSocialPost() {
+
+  const post = {
+    users:['https://images.unsplash.com/photo-1698296449732-d0a3195f147a?auto=format&fit=crop&q=80&w=3087&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D', 'https://images.unsplash.com/photo-1686040054953-9cb42085ca9c?auto=format&fit=crop&q=80&w=3087&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D', 'https://images.unsplash.com/photo-1697917225161-f43b4e60d2f4?auto=format&fit=crop&q=80&w=1811&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D'],
+    likes: 47,
+    info:{
+      comments: 10,
+      shared:2
+    }
+  }
+  return (
+    <Card
+      variant="plain"
+      sx={(theme) => ({
+        boxShadow: theme.shadow.md,
+        borderRadius: "1.5rem",
+        background: "white"
```

Comment:
```suggestion
        background: "background.body"
```
This is better because it will work for both light and dark mode.